pkg/agent/openflow/regmarks.go

```diff
@@ -0,0 +1,105 @@
+package openflow
+
+import binding "antrea.io/antrea/pkg/ovs/openflow"
+
+// Fields used reg.
+var (
+	// reg0 (NXM_NX_REG0)
+	// reg0[0..3]: Field to mark the packet source. Marks in this field include,
+	//   - 0: from the tunnel port
+	//   - 1: from antrea-gw0
+	//   - 2: from the local Pods
+	//   - 4: from the Bridge interface
+	//   - 5: from the uplink interface
+	PktSourceField  = binding.NewRegField(0, 0, 3, "PacketSource")
+	FromTunnelMark  = binding.NewRegMarkWithField(PktSourceField, 0)
```

Comment:
If there is no NewRegMarkWithoutField, maybe just name it NewRegMark to be simple?